bin/serve.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import { argv } from 'yargs';
+import { existsSync, readFileSync } from 'fs'
+import { TestxServer } from '../src';
+
+async function startServer(model: string) {
+    const server = new TestxServer({
+        schema: model
+    });
+
+    // start the server
+    await server.start();
+    // retrieve the server url
+    console.log(`Running on ${await server.httpUrl()}`);
+}
+
+
+if (argv.model) {
+    const modelLocation = argv.model as string;
+    if (existsSync(modelLocation)) {
+        const fileContent = readFileSync(modelLocation, { encoding: 'utf8' });
+        startServer(fileContent);
```

Comment:
```suggestion
        startServer(fileContent).catch(e => throw e);
```
an alternative would be to log the error and stack and then exit with something different from 0, the only problem with the current way is that nodejs will give an error like `not handled exception in promise` which is not so nice

Reply:
the best approach will be to create a `main` function
```
async function main() {

}

main().catch(e => {throw e});
```